### PR TITLE
kobaltlabInc.HashPhotosTransfer version 1.0.6.14

### PR DESCRIPTION
**Pull request by MysticCereal**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100097)

### CODE REVIEW

**Comment by MysticCereal** [comment]
@microsoft-github-policy-service agree

**Comment by MysticCereal** [comment]
@microsoft-github-policy-service agree

**Comment by MysticCereal** [comment]
Publisher URL: https://www.hashphotos.app/
Installer URL: https://www.hashphotos.app/#hashphotostransfer
As noted at the bottom of the webpage, "© 2023 kobaltlab Inc."

File: manifests/k/kobaltlabInc/HashPhotosTransfer/1.0.6.14/kobaltlabInc.HashPhotosTransfer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: kobaltlabInc.HashPhotosTransfer
+PackageVersion: 1.0.6.14
+PackageLocale: en-US
+Publisher: kobaltlab Inc.
+PackageName: HashPhotos Transfer
```
Comment (stephengillie) [inline manifests/k/kobaltlabInc/HashPhotosTransfer/1.0.6.14/kobaltlabInc.HashPhotosTransfer.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.hashphotos.app/
PrivacyUrl:   https://www.hashphotos.app/privacypolicy
PackageName: HashPhotos Transfer
PackageUrl: https://www.hashphotos.app/#hashphotostransfer
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**